src/arrayUtils.js

```diff
@@ -10,7 +10,6 @@ function intArrayFromString(stringy, dontAddNull, length) {
 }
 
 // Temporarily duplicating function pending Python preprocessor support
-var ASSERTIONS;
```

Comment:
Is this because we start to unconditionally print `var ASSERTIONS` later down in this PR?

Reply:
Yes. Under some circumstances `ASSERTIONS` was defined twice and Closure Compiler complained. So I've made sure it is always defined exactly once. Closure Compiler can remove it from source code later if not used at all.